src/AppInstallerCommonCore/Public/winget/Resources.h

```diff
@@ -20,6 +21,16 @@ namespace AppInstaller
         struct StringId : public std::wstring_view
         {
             explicit constexpr StringId(std::wstring_view id) : std::wstring_view(id) {}
+
+            // Sets the placeholder values in the resolved string id.
+            template<typename ...T>
+            Utility::LocIndString operator()(T ... args) const;
+
+            friend std::ostream& operator<<(std::ostream& out, StringId si);

```

Comment:
`friend` is not needed here, removed keyword and moved function directly under the same namespace.
`Resolve` is private to ensure we always use the new syntax (`operator()`) which calls the function internally (if the resolved string id expects arguments, the new syntax will `static_assert` the types to prevent user errors).